src/backend/dx11/src/conv.rs

```diff
@@ -730,6 +730,15 @@ pub(crate) fn map_depth_stencil_desc(
         None => unsafe { mem::zeroed() },
     };
 
+    let stencil_read_only = write_mask == 0 ||
```

Comment:
I wonder if we should make those conditions even more precise (at some point). I.e. technically speaking, if your stencil function is ALWAYS, then it doesn't matter what `StencilFailOp` is for any of the sides.

Reply:
Good point, shouldn't be too hard to add.